src/NuGet.Clients/NuGet.PackageManagement.UI/Options.cs

```diff
@@ -9,9 +9,9 @@
 
 namespace NuGet.PackageManagement.UI
 {
-    public class Options : INotifyPropertyChanged
```

Comment:
Had to change the name because of a conflict with the options folder in NuGet.PackageManagement.UI